docs/config.rst

```diff
@@ -572,6 +572,16 @@ Features
    .. _RVM: https://rvm.io/
    .. _RBENV: https://github.com/rbenv/rbenv
 
+.. attribute:: LP_ENABLE_TERRAFORM
+   :type: bool
+   :value: 1
+
+   Display the currently activated Terraform workspace.
```

Comment:
This won't use the below link without backticks
```suggestion
   Display the currently activated `Terraform`_ workspace.
```
You can see that in the built docs: https://liquidprompt--669.org.readthedocs.build/en/669/config.html#LP_ENABLE_TERRAFORM